drivers/hmis/app/graphql/types/base_enum.rb

```diff
@@ -20,6 +20,10 @@ def self.with_enum_map(enum_map, prefix: '', prefix_description_with_key: true)
         member_values[:key] = "#{prefix}#{member[:key]}"
         member_values[:desc] = prefix_description_with_key ? "(#{member_values[:value]}) #{member_values[:desc]}" : member_values[:desc]
         member_values = yield member if block_given?
+
+        # Ensure we are using DATA_NOT_COLLECTED key for 99s
+        member_values[:key] = 'DATA_NOT_COLLECTED' if member_values[:value] == 99
```

Comment:
Are there potentially numbs with a valid value of 99 that isn't DNC?  I don't think so at the moment, but I could see that coming up later.

Reply:
Hmm, not currently. This only gets run during codegen, so we should be able to notice an incorrect change in the schema if that happens.